### PR TITLE
Add ign-plugin v1.1.0

### PR DESCRIPTION
**Pull request by seanyen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by seanyen** [comment]
cc @wolfv

File: recipes/libignition-physics/bldpkg.bat [hunk 1]
```diff
@@ -0,0 +1,20 @@
+:: MSVC is preferred.
+set CC=cl.exe
+set CXX=cl.exe
```
Comment (scopatz) [inline recipes/libignition-physics/bldpkg.bat]: This doesn't seem right

Comment (seanyen) [reply recipes/libignition-physics/bldpkg.bat]: @scopatz I am trying to use MSVC/Ninja combination and if I don't do this, the Ninja generator will pick up `g++` opposing to `MSVC` (it seems the one is added to `Chocolatey\bin` folder and which is in the `%PATH%` search path), so I had to add this hints to make CMake pick up the desired toolchain. Let me know if there is a better recipe doing that.

Comment (scopatz) [reply recipes/libignition-physics/bldpkg.bat]: CC @conda-forge/staged-recipes @isuruf

**Pull request merged**